arrow/arrow.py

```diff
@@ -575,6 +594,32 @@ def float_timestamp(self):
 
         return self.timestamp + float(self.microsecond) / 1000000
 
+    @property
+    def fold(self):
+        """ Gets the ``fold`` value of the :class:`Arrow <arrow.arrow.Arrow>` object. """
+
+        # in python < 3.6 _datetime will be a _DatetimeWithFold if fold=1 and a datetime with no fold attribute
+        # otherwise, so we need to return zero to cover the latter case
+        return getattr(self._datetime, "fold", 0)
+
+    @fold.setter
+    def fold(self, val):
```

Comment:
Are `Arrow` objects supposed to be mutable? I don't see properties for other attributes like `year` or `month`, so I guess they are technically mutable, but wouldn't they cause the `._datetime` attribute to get out of sync with the `arrow.Arrow` value?

Also, it looks like this object is hashable! I think you should probably remove the setter here (and presumably remove the other setters and make this a properly immutable object).